docs/documentation/platform/secret-rotation/mysql.mdx

```diff
@@ -3,35 +3,98 @@ title: "MySQL/MariaDB"
 description: "Rotated database user password of a MySQL or MariaDB"
 ---
 
-Infisical will update periodically the provided database user's password.
+Infisical MySQL secret rotation allows you to automatically rotate your MySQL database user's password at regular intervals.
 
 <Warning>
 	At present Infisical do require access to your database. We will soon be released Infisical agent based rotation which would help you rotate without direct database access from Infisical cloud.
 </Warning>
 
+## Prerequisite
+
+1. Create two users with the required permission in your MySQL instance. Let's call them `user-a` and `user-b`.
+2. Create another MySQL user with permission to update the passwords of `user-a` and `user-b`. Let's call it `admin`
+
+To learn more about MySQL Permissioning, please follow this [documentation](https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html).
+
 ## Working
 
-1. User's has to create the two user's for Infisical to rotate and provide them required database access
-2. Infisical will connect with your database with admin access
-3. If last rotated one was username1, then username2 is chosen to be rotated
-5. Update it's password with random value
-6. After testing it gets saved to the provided secret mapping
+1. Infisical connects to your database using the provided `admin` user account.
```

Comment:
Please the steps component https://mintlify.com/docs/content/components/steps

Reply:
@maidul98 Steps is better when user need to follow it one by one. The working is just a set of points on how the system does it. 

Also steps without a description is not a good thing. The above points are short and precise right